### PR TITLE
GH-8993 added flow types to files js

### PR DESCRIPTION
**Pull request by pradeepmurugesan**
#### Summary
added flow types to files.js

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/8993

### CODE REVIEW

File: src/actions/files.js [hunk 1]
```diff
@@ -52,9 +54,10 @@ export function getMissingFilesForPost(postId) {
     };
 }
 
-export function uploadFile(channelId, rootId, clientIds, fileFormData, formBoundary) {
-    return async (dispatch, getState) => {
-        dispatch({type: FileTypes.UPLOAD_FILES_REQUEST}, getState);
+export function uploadFile(channelId: string, rootId: string, clientIds: Array<String>,
+    fileFormData: any, formBoundary: string) {
```
[the remark this answers is not in the document]
Comment (pradeepmurugesan) [reply src/actions/files.js]: @jespino changing it to fileFormData: File works.. But i am wondering where the File type is defined.. When I cmd+clicked and navigated to the definition I landed in lib.dom.d.ts. 

Is this the right type ?

[the remark this answers is not in the document]
Comment (jespino) [reply src/actions/files.js]: Yes, it is. Is the standard DOM `File` type (normally part of a `FileList` object, which is in the `files` attribute of `HTMLInputElement`).

[the remark this answers is not in the document]
Comment (pradeepmurugesan) [reply src/actions/files.js]: Cool.. Thanks.. Fixed the same..

**Comment by jespino** [comment]
Thank you! :):):):):)

**Pull request merged**